DesignMotivations.md

```diff
@@ -1,22 +1,31 @@
-## Roles
+## Roles and epics
 
 #### QM software developer
-I write code that performs QM calculations and produces output for its end users
+Role: I write code that performs QM calculations and produces output for its end users
```

Comment:
Do we want to classify end users slightly? This can mean anything from a organic bench chemist to a QM power user.